### PR TITLE
Added Options Form

### PR DESCRIPTION
**Pull request by ThiconZ**
Sorted command lists in alphabetical order
Added temp options command to open the form, this will be changed to the
minimap button
Options form still needs values updated to match the changed commands

### CODE REVIEW

File: !Questie/QuestieOptionsForm.lua [hunk 1]
```diff
@@ -233,6 +233,9 @@ function Questie:OptionsForm_SettingOnEnter(SettingsName)
     elseif(SettingsName == "MaxShowLevel") then
         QuestieOptionsToolTip:AddLine("Hides quests until <X> levels above players level (default=3)", 1, 1, 0)
 
+    elseif(SettingsName == "MinimapButton") then
+        QuestieOptionsToolTip:AddLine("Removes questie minimap button (default=true)", 1, 1, 0)
```
Comment (Muehe) [inline !Questie/QuestieOptionsForm.lua]: This message should read `"Show Questie minimap button (default=true)"`. Currently it sounds like the button is turned off by default.

File: !Questie/Modules/QuestieGUI.lua [hunk 2]
```diff
@@ -26,22 +26,16 @@ Questie.minimapButton:SetHighlightTexture('Interface\\Minimap\\UI-Minimap-ZoomBu
 Questie.minimapButton:SetPoint("TOPLEFT", Minimap, "TOPLEFT", 52-(80*cos(QuestieMinimapPosition)),(80*sin(QuestieMinimapPosition))-52)
 Questie.minimapButton:SetScript("OnClick", function()
     if ( arg1 == "LeftButton" ) then
-        Questie:Toggle()
+      Questie:OptionsForm_Display()
```
Comment (Muehe) [inline !Questie/Modules/QuestieGUI.lua]: Indents are 4 spaces, not 2.

File: !Questie/Modules/QuestieGUI.lua [hunk 3]
```diff
@@ -26,22 +26,16 @@ Questie.minimapButton:SetHighlightTexture('Interface\\Minimap\\UI-Minimap-ZoomBu
...
     end
     if (arg1 == "RightButton") then
-        QuestieConfig.hideMinimapIcons = not QuestieConfig.hideMinimapIcons
-        if QuestieConfig.hideMinimapIcons then
-            DEFAULT_CHAT_FRAME:AddMessage("QuestieStarters:|c0000ffc0 (Are now being hidden) |r")
-        else
-            DEFAULT_CHAT_FRAME:AddMessage("QuestieStarters:|c0000ffc0 (Are now being shown) |r")
-        end
-        Questie:AddEvent("DRAWNOTES", 0.1);
+      Questie:Toggle()
```
Comment (Muehe) [inline !Questie/Modules/QuestieGUI.lua]: Same here.

**Comment by ThiconZ** [comment]
image
Clicking the X button to close the Form will cancel any changes made.

**Comment by Muehe** [comment]
Merged.  :+1:

**Pull request closed without merging**